Documentation/Contributors/BuildGuide/README.md

```diff
@@ -1,10 +1,42 @@
 # Build Guide
 
+- [Quickstart](#quickstart)
 - [Get the Code](#get-the-code)
 - [Build the Code](#build-the-code)
 - [Build Scripts](#build-scripts)
 - [Travis and Continuous Integration](#travis-and-continuous-integration)
 
+## Quickstart
+
+1. Get the code.
```

Comment:
Since this is the quick start, I would suggest making this section even more brief, but linking to sections below for more details. For example:


1. [Clone the repo](#get-the-code)
2. [Build the code](#build-the-code) and start the server. At the `cesium` root, run:
````
npm install
npm build
npm start
````
3. In a browser, navigate to [`http://localhost:8080/`](http://localhost:8080/)